dev/devicelab/manifest.yaml

```diff
@@ -1,349 +0,0 @@
-#
-# Required properties:
-#
-# * description: free form string
-#       describes what the task is about
-# * stage: one of "devicelab", "devicelab_ios", "devicelab_win"
-#       tasks are grouped by stage so they appear next to each on the dashboard and have a distinct
-#       icon attached to them. Stages can be used to create dependencies, e.g. we may decide to not
-#       run device tests on red Cirrus builds.
-# * required_agent_capabilities: a list of strings
-#       list of capabilities a devicelab agent must have to be able to run this task.
-#
-# Optional properties:
-#
-# * flaky: boolean true or false
-#       whether the task is considered flaky; the result of running a flaky task does not affect
-#       the overall build status. Tests should be marked flaky when newly added, until they have
-#       been proved for a few cycles. Tests that are actually flaky but not being actively worked
-#       on should be hidden because they just cause confusion otherwise.
-# * on_luci: boolean true or false
-#       whether the task runs on LUCI builders. Tests will be gradually marked on_luci during the
-#       devicelab to LUCI migration. And when the migration is complete, on_luci flag will be
-#       removed.
-
-tasks:
-  # Tests of compiling in a variety of modes
-
-  complex_layout_ios__compile:
-    description: >
-      Collects various performance metrics of compiling the Complex
-      Layout sample app for iOS from Mac.
-    stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios"]
-
-  complex_layout_win__compile:
-    description: >
-      Collects various performance metrics of compiling the Complex
-      Layout for Android from Windows.
-    stage: devicelab_win
-    required_agent_capabilities: ["windows/android"]
-
-  basic_material_app_ios__compile:
-    description: >
-      Collects various performance metrics of compiling the default
-      app for iOS from Mac.
-    stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios"]
-
-  basic_material_app_win__compile:
-    description: >
-      Collects various performance metrics of compiling the default
-      app for Android from Windows.
-    stage: devicelab_win
-    required_agent_capabilities: ["windows/android"]
-
-  flutter_gallery_ios__compile:
-    description: >
-      Collects various performance metrics of compiling the Flutter
-      Gallery for iOS from Mac.
-    stage: devicelab_ios
-    required_agent_capabilities: ["mac/ios"]
-
-  flutter_gallery_win__compile:
-    description: >
-      Collects various performance metrics of compiling the Flutter
-      Gallery for Android from Windows.
-    stage: devicelab_win
-    required_agent_capabilities: ["windows/android"]
-
-  windows_chrome_dev_mode:
-    description: >
-      Run flutter web on the devicelab and hot restart.
-    stage: devicelab_win
-    required_agent_capabilities: ["windows/android"]
-
-  # iOS on-device tests
-
-  flutter_gallery_sksl_warmup__transition_perf_e2e_ios32:
```

Comment:
There's various iOS tests missing, can you add those as well?

Reply:
I assume those are not to be run on this branch?

Reply:
They should be. These were missed from the devicelab migration